smb_path_tracker/package.xml

```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0"?>
+<package format="2">
+  <name>smb_path_tracker</name>
+  <version>0.0.0</version>
+  <description>Path tracker for SuperMegaBot</description>
```

Comment:
Could you add you (or@nospelt?) as maintainer and define the original author as author? 
Somehow, this seems like plagiarism to me.